src/docx/assets/zarzadzenie/word/document.xml

```diff
@@ -0,0 +1,150 @@
+<?xml version='1.0' encoding='UTF-8' standalone='yes'?>
+<w:document xmlns:ve="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+  xmlns:o="urn:schemas-microsoft-com:office:office" 
+  xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships" 
+  xmlns:m="http://schemas.openxmlformats.org/officeDocument/2006/math" 
+  xmlns:v="urn:schemas-microsoft-com:vml" 
+  xmlns:wp="http://schemas.openxmlformats.org/drawingml/2006/wordprocessingDrawing" 
+  xmlns:w10="urn:schemas-microsoft-com:office:word" 
+  xmlns:w="http://schemas.openxmlformats.org/wordprocessingml/2006/main" 
+  xmlns:wne="http://schemas.microsoft.com/office/word/2006/wordml"
+  xmlns:pic="http://schemas.openxmlformats.org/drawingml/2006/picture">
+  <w:body>
+    <w:p>
+      <w:pPr>
+        <w:pStyle w:val="centerAlignedTitleGreenColor"/>
+      </w:pPr>
+      <w:r>
+        <w:t>Poradnia Psychologiczno – Pedagogiczna we Wrześni</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:spacing w:before="100"/>
+        <w:pStyle w:val="Uppercase"/>
+      </w:pPr>
+      <w:r>
+        <w:t>PPP</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:pStyle w:val="City-and-date"/>
+      </w:pPr>
+      <w:r>
+        <w:t>{{ city }}, {{ date }}</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:spacing w:before="500"/>
+        <w:pStyle w:val="centerHeader"/>
+      </w:pPr>
+      <w:r>
+        <w:t>Zarządzenie nr {{ no }}/2020</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:spacing w:before="50"/>
+        <w:pStyle w:val="centerHeader"/>
+      </w:pPr>
+      <w:r>
+        <w:t>dyrektora Poradni z dnia {{ date }} r.</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:spacing w:before="50"/>
+        <w:pStyle w:val="centerHeader"/>
+      </w:pPr>
+      <w:r>
+        <w:t>w sprawie powołania zespołu orzekającego</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:spacing w:before="50"/>
+        <w:pStyle w:val="centerHeader"/>
+      </w:pPr>
+      <w:r>
+        <w:t>§1</w:t>
+      </w:r>
+    </w:p>
+    <w:p>
+      <w:pPr>
+        <w:spacing w:before="50"/>
+        <w:pStyle w:val="normalParagraph"/>
+      </w:pPr>
+      <w:r>
+        <w:t xml:space="preserve">Dyrektor Poradni Psychologicznej w {{ supportCenter.city }}, {{ supportCenter.post }}, {{ supportCenter.address }}, na podstawie § 4 ust. 1-6 rozporządzenia Ministra Edukacji Narodowej z dnia 7 września 2017 r. w sprawie orzeczeń i opinii wydawanych przez zespoły orzekające działające w publicznych poradniach psychologiczno – pedagogicznych, (Dz. U. z 2017 r. poz. 1743) powołuje zespół orzekający do rozpatrzenia wniosku </w:t>
```

Comment:
I think that use whole name of support center ("Poradnia Psychologiczno - Pedagogiczna we Wrześni", "Poradnii Psychologiczno - Pedagogicznej we Wrześni") would be better. First, it allow you to ommit declension of cities or changes of conjunctions (**we** Wrześni but **w** Warszawie)

Reply:
I'm afraid we don't have just a name of support center such as _Poradnia Psychologiczno - Pedagogiczna we Wrześni_. In this case, there is _Zespół Orzekający przy Poradni Psychologiczno-Pedagogicznej w Grodzisku Wlkp._. <br>
I take the view there is a need to add such kind of information in form.

Reply:
I think, that information such name of support center with its all forms (nominative, genitive...) should be set as some kind of settings. Now we can set them as fixture. I can see benefits of separating _Zespół orzekający przy_ and _Poradnii Psychologiczno - Pedagogicznej_. The former part is a fixed one, and we can just hardcoded it.  I think that It'd be resonable to add that fixture (eg. `nameNominative`, `nameGenetive`) and use it in that place.

Reply:
I've hardcoded the support center name as you suggested above.